modules/stanford_publication_views_reference/css/stanford_publication_reference.css

```diff
@@ -0,0 +1,15 @@
+/* Styles for the Stanford Publication Feature */
+
+/* Field spacing */
+
+.view-stanford-publications-reference .views-field-field-s-image-info {
+	margin-bottom: 10px;
```

Comment:
@josephgknox, You should be using spaces and not tabs in your indenting. The convention is two spaces. You can set your editor to use two spaces when you hit the tab button. I will merge but please update your editor.